glotaran/project/result.py

```diff
@@ -178,6 +184,8 @@ def markdown(self, with_model: bool = True, base_heading_level: int = 1) -> Mark
             If `True`, the model will be printed with initial and optimized parameters filled in.
         base_heading_level : int
             The level of the base heading.
+        model_is_detail: bool
```

Comment:
Uff, naming thing. I think what you mean is "render model as detail" or "model as detail" for short, maybe "model in detail" or "wrap model in detail". Anything with is seems more like a check.

Reply:
How about `wrap_model_in_details`, I guess that is most descriptive in respect to what it does

Reply:
Resolved in [fd99c00](https://github.com/glotaran/pyglotaran/pull/1098/commits/fd99c0009e9112b31ebef55f7db5f1a8d79c6508)